index.html

```diff
@@ -7,21 +7,83 @@
   </head>
 
   <body>
+
+    <div class="page-container hide" id="/#!/user-info-edit">
```

Comment:
```suggestion
    <div class="page-container hide" data-id="/#!/user-info-edit">
```
id보다 dataset을 이용하는 편이 더 나을 것 같아요~ id 명으로 이러한 /#! 기호를 쓰지 않아서 처음 봤을 때 어색하게 느껴졌네요.

Reply:
아 알고 있는 부분이었는데도 이상하게 dataset을 활용할 생각을 하지 못했네요 ㅎㅎ감사합니다!